### PR TITLE
Adds the english h1-h6 element

### PR DESCRIPTION
**Pull request by kathyoleary**
:green_book:  Ready to go!  Need two thumbs please!

### CODE REVIEW

**Comment by bigtiger** [comment]
:+1:

**Comment by savance-ford** [comment]
:+1:

File: elements/h1-h6.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+  </header>
+  <div class="definition">
+    <div class="definition-box definition-spanish">
+      <h4>Definici&oacute;n</h4>
+      <p></p>
+    </div>
+    <div class="definition-box definition-english">
+      <h4>Definition</h4>
+      <p>Not to be confused with the &lt;header&gt; element which is a whole section of a web page: heading elements are used to describe the topic or give the title of the section it is introducing.  There are six graduated levels of font size and boldness with &lt;h1&gt; being the largest and boldest down to &lt;h6&gt; as the smallest and least boldface.</p>
```
Comment (ericanaughton) [inline elements/h1-h6.html]: the colon should be a comma or semi-colon here

File: elements/h1-h6.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+  </div>
+  <div class="how-to-remember">
+    <div class="how-to-remember-box how-to-remember-spanish">
+      <h4>&iquest;C&oacute;mo recordar?</h4>
+      <p></p>
+    </div>
+    <div class="how-to-remember-box how-to-remember-english">
+      <h4>How to Remember</h4>
+      <p>In a concert, the number one artist headlining in the show is the biggest and brightest, the second in importance is slightly smaller and under the headliner, and so on down the line - just as heading level 1 &lt;h1&gt; is the biggest and boldest, heading level 2 &lt;h2&gt; is slightly smaller and less bold, and so on down the line.</p>
```
Comment (ericanaughton) [inline elements/h1-h6.html]: for graduated, did you mean to say graduate, or gradual?

Comment (ericanaughton) [inline elements/h1-h6.html]: this is a run on sentence.
you could try something like this:
"In a concert, the number one artist headlining in the show is the biggest and brightest. The second in importance is slightly smaller and under the headliner, and so on down the line. Just as heading level 1 `<h1>` is the biggest and boldest, heading level 2 `<h2>` is slightly smaller and less bold."

**Comment by kathyoleary** [comment]
graduated should be the correct term- at least according to Merriam-Webster

Examples of graduated in a sentence
a series of bowls in graduated sizes
http://www.merriam-webster.com/dictionary/graduated

Does it sound off?

**Comment by ericanaughton** [comment]
im just not familiar with the word used that way, learn something new everyday! :+1:

**Pull request merged**